hack/windows/Install-OVS.ps1

```diff
@@ -136,17 +141,17 @@ function InstallOVS() {
     $OVSDriverDir = "$OVSInstallDir\driver"
 
     # Install OVS driver certificate.
-    if (Test-Path $OVSDriverDir\package.cer) {
+    $DriverFile="$OVSDriverDir\OVSExt.sys"
```

Comment:
I see a reference to `$OVSInstallDir\driver\ovsext.sys` later in the file. Is it the same one? Aren't path case sensitive on Windows, why the discrepancy?

Reply:
I tested on a Windows VM. It turns out no.
```
PS C:\Users\Administrator\tmp2> ls .\openvswitch\driver\


    Directory: C:\Users\Administrator\tmp2\openvswitch\driver


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------        3/16/2021   8:44 PM             72 install.cmd
------        3/16/2021   8:44 PM          10032 ovsext.cat
------        3/16/2021   8:44 PM           2359 ovsext.inf
------        3/16/2021   8:44 PM         196536 OVSExt.sys
------        3/16/2021   8:44 PM             18 uninstall.cmd


PS C:\Users\Administrator\tmp2> ls .\openvswitch\driver\OVSExt.sys


    Directory: C:\Users\Administrator\tmp2\openvswitch\driver


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------        3/16/2021   8:44 PM         196536 OVSExt.sys


PS C:\Users\Administrator\tmp2> ls .\openvswitch\driver\ovsext.sys


    Directory: C:\Users\Administrator\tmp2\openvswitch\driver


Mode                LastWriteTime         Length Name
----                -------------         ------ ----
------        3/16/2021   8:44 PM         196536 ovsext.sys
```

Reply:
surprising, thanks for checking

Reply:
Path is not case sensitive in Windows.